src/Playwright.Tests/NetworkPostDataTests.cs

```diff
@@ -47,7 +47,7 @@ public async Task ShouldReturnCorrectPostdataBufferForUtf8Body()
 
             var request = task.Result;
             Assert.Equal(expectedJsonValue, request.PostData);
-            Assert.Equal(value, request.PostDataJSON().RootElement.GetString());
+            Assert.Equal(value, request.PostDataJSON()?.GetString());
```

Comment:
we know we should get a value there right?

```suggestion
            Assert.Equal(value, request.PostDataJSON().GetString());
```

Reply:
It does not compile without ? because the value is nullable.